pkgdb/src/realisepkgs/realisepkgs-lockfile.cc

```diff
@@ -55,12 +55,14 @@ RealisepkgsLockfile::from_v0_content( const nlohmann::json & jfrom )
                 lockedPackage->input.attrs );
               input.url = nix::FlakeRef::fromAttrs( input.attrs ).to_string();
 
-              this->packages.emplace_back(
-                RealisepkgsLockedPackage { system,
-                                           installId,
-                                           input,
-                                           lockedPackage->attrPath,
-                                           lockedPackage->priority } );
+              this->packages.emplace_back( RealisepkgsLockedPackage {
+                system,
+                installId,
+                input,
+                lockedPackage->attrPath,
+                lockedPackage->priority,
+                // TODO - this is a hack, not supported in V0?
+                OutputsToOutpaths() } );
```

Comment:
Do we need to continue supporting V0, or can we dump this and error out at this point as part of a deprecation strategy?

Reply:
I see this was just deprecated so I'll just leave this and not worry much about testing these paths.